gems/aws-sdk-s3/lib/aws-sdk-s3/customizations/object.rb

```diff
@@ -320,29 +328,31 @@ def upload_file(source, options = {})
       #     # and the parts are downloaded in parallel
       #     obj.download_file('/path/to/very_large_file')
       #
-      # @param [String] destination Where to download the file to
+      # @param [String] destination Where to download the file to.
       #
       # @option options [String] mode `auto`, `single_request`, `get_range`
       #  `single_request` mode forces only 1 GET request is made in download,
       #  `get_range` mode allows `chunk_size` parameter to configured in
       #  customizing each range size in multipart_download,
       #  By default, `auto` mode is enabled, which performs multipart_download
       #
-      # @option options [String] chunk_size required in get_range mode
+      # @option options [String] chunk_size required in get_range mode.
       #
-      # @option options [String] thread_count Customize threads used in multipart
-      #   download, if not provided, 10 is default value
+      # @option options [String] thread_count (10) Customize threads used in the
```

Comment:
still have string here?

Reply:
Yeah - I changed it (locally) as well as some other stuff. I will update soon.